docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -0,0 +1,597 @@
+Re’s insured population from the weighted blend of Chinese provinces. Forecasts 
+run from year 2020 to 2040.
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+.. todo::
+
+ add more literature background
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+IHME will estimate the yearly number of cases of benign and invasive cervical 
+cancer detected under specific cervical cancer screening practices and the yearly 
+number of deaths from undetected cervical cancer (both in unit of per 100,000 
+insured person-years) in order to identify pay-out trends for cervical cancer 
+claims.
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal variables
+--------------------
+ 
+Outcome
+ - Cervical cancer detection (benign and invasive)
+ - Mortality and morbidity of cervical cancer
+Most proximal determinant/exposure
+ - Cervical cancer status
+ - Cervical cancer screening
+ - HPV infection  
+Confounders
+ - age
+ - sex
+Effect modifiers
+ - N/A
+Mediators
+ - N/A
+
+
+.. _4.0:
+
+4.0 Intervention
+++++++++++++++++
+Scale-up of both cervical cancer screening coverage and HPV vaccination rate 
+among insured population
+
+.. _4.1:
+
+4.1 Simulation scenarios
+------------------------
+**Baseline:** project existing level of HPV vaccination for women aged 15 to 44 
+and cervical cancer screening for women aged 25 to 64, where:
+ - Women without HPV infection are provided with cytology every three years
+ - Women with HPV infection are provided cytology every year
+
+**Alternative scenario:** by 2040, linear ramp up both HPV vaccination to cover 
+90% of the women aged 15 to 44 in 2021-2025 and cervical cancer screening to cover 
+70% of women aged 25 to 64 in 2021-2030, where:
+ - Women without HPV infection are provided with cytology every three years
+ - Women with HPV infection are provided cytology every year
+
+.. image:: cervical_cancer_scale_up.PNG
+
+.. list-table:: Intervention scale-up
+   :header-rows: 1
+
+   * - Scenario
+     - Intervention
+     - Year
+     - Coverage
+   * - Baseline
+     - Cervical cancer screening
+     - 2020-2040
+     - 20%
+   * - Alternative
+     - Cervical cancer screening
+     - 2020-2030
+     - Stay 20% in 2020-2021 then linearly ramp up from 20% to 70% in 2021-2030
+   * - Alternative
+     - Cervical cancer screening
+     - 2031-2040
+     - 70%
+   * - Baseline
+     - HPV vaccination
+     - 2020-2040
+     - 10% (stand-in value)
+   * - Alternative
+     - HPV vaccination
+     - 2020-2030
+     - Stay 10% in 2020-2021 then linearly ramp up from 10% to 90% in 2021-2030
+   * - Alternative
+     - HPV vaccination
+     - 2031-2040
+     - 90%
+
+.. note::
+
+ - Initial screening coverage = 20.7% with 95%CI 18.6-22.8 (Wang et al. 2015). 
+ - No data identified for current HPV vaccination rates in China.
+ - The target screening and HPV vaccination coverage are guided by WHO cervical 
+   cancer elimination strategy.
+
```

Comment:
For initial cervical cancer screening coverage do you want to use 20% or 20.7%? I would suggest keeping the number consistent between the table and note!

Reply:
20.7% was found from literature, the initial screening coverage we could use based on the evidence is an arbitrary number close to or greater than 20.7%, 20%-30% could be a reasonable range.